### PR TITLE
Log stream

### PR DESCRIPTION
**Pull request by chris-olszewski**
Functionally everything is here.

Premerge tasks:
- [x] verify that cli options work as intended
- [x] clean up comments
- [x] update docs

### CODE REVIEW

File: src/logs/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export default function (opts) {
+  const functionName = genName(opts.name).fullName
+  const aliasName = opts.stage
+  const stream = opts.stream
+  const region = opts.region || pkg().shep.region
+
+  AWS.config.update({ region })
+
+  // Should gracefully handle if function doesn't exist
+  return Promise.all([getLogGroup({ functionName }), getAliasVersion({ functionName, aliasName })])
```
Comment (southpolesteve) [inline src/logs/index.js]: Check out `Promise.join` here. It basically does what you are doing with `all` and `spread` but in one command

File: src/logs/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  .spread((logGroupName, functionVersion) => getLogs({ logGroupName, functionVersion, stream }))
+  .then(printEventsLoop)
+}
+
+function printEventsLoop ({ events, nextLogCall }) {
+  const rate = 250 // note this can't go under 200 as we're limited to 5 requests/sec
```
Comment (southpolesteve) [inline src/logs/index.js]: Let's make this 500 out of the gate. Some AWS API limits are across the entire account, so would like to have a bigger buffer.

**Comment by chris-olszewski** [comment]
I think everything is set to merge from my end

**Comment by southpolesteve** [comment]
Looks good to me. Merge away. I think this will be a minor release since we are adding new stuff. So 2.1?

**Pull request merged**